pkg/azure/actuator_test.go

```diff
@@ -190,32 +173,6 @@ func TestDecodeToUnknown(t *testing.T) {
 	}
 }
 
-func TestAnnotations(t *testing.T) {
```

Comment:
This test did nothing to my eyes, it only ran the constructor. I wasn't sure how to adapt it.